onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -191,6 +191,25 @@ void RegisterNchwcSchemas() {
       .FillUsing(NchwcGlobalPoolOpSchemaGenerator);
 }
 
+void RegisterBertSchemas() {
+
+  ONNX_CONTRIB_OPERATOR_SCHEMA(Attention)
+      .SetDomain(kOnnxDomain)
+      .SinceVersion(1)
+      .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
```

Comment:
Maybe I'm missing something, but it seems to me that `Attention` is not an ONNX operator supported since version 1. Shouldn't this be in a different domain to avoid conflict with possible future ONNX operators?

Reply:
@TMVector, Thank for suggestion, I'll move it to kMSDomain.